base/utils/R/get.analysis.filenames.r

```diff
@@ -1,19 +1,27 @@
 ##' Generate ensemble filenames
-##' 
+##'
 ##' @name ensemble.filename
 ##' @title Generate ensemble filenames
```

Comment:
@infotroph removing @name <> and @title <> won't cause any troubles in the documentation files?

Reply:
Nope, because Roxygen defaults to using the function name for @name and the first line of the comment block for @title, and both of those are identical here so it's cleaner to avoid duplicating.

Reply:
yep got it!

Reply:
(You can verify this by looking at the generated `man/ensemble.filename.Rd` before and after making the change -- the `\name{..}` and `\title{...}` lines shouldn't change.)